repo/meta/schema/package-schema.json

```diff
@@ -65,6 +65,13 @@
           "version"
         ]
       }
+    },
+    "tags": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "pattern": "[a-z]+"
```

Comment:
why the restriction of only lower case letters? For example users want to tag something as `python3`.